apps/mobile/src/hooks/useRegisterForNotifications.ts

```diff
@@ -49,9 +50,27 @@ const useRegisterForNotifications = (): NotificationsProps => {
   const { storePrivateKey, getPrivateKey } = useSign()
   // Redux
   const dispatch = useAppDispatch()
-  const activeSafe = useDefinedActiveSafe()
+  const activeSafe = useAppSelector(selectActiveSafe)
   const delegatedAccounts = useAppSelector(selectDelegatedAccounts)
-  const { ownerFound, accountType } = useNotificationGTWPermissions().getAccountType()
+
+  const glob = useGlobalSearchParams<{ safeAddress?: string; chainId?: string; import_safe?: string }>()
+
+  if (!glob) {
+    throw new Error('useRegisterForNotifications :: glob is undefined')
+  }
```

Comment:
Why is it necessary to throw here? Few lines later you set the glob safeAddress to whatever was in the activeSafe?